java-manta-client-unshaded/src/main/java/com/joyent/manta/client/multipart/MantaMultipartUploadTuple.java

```diff
@@ -78,16 +78,15 @@ public int compareTo(final MantaMultipartUploadTuple that) {
     }
 
     @Override
-    public boolean equals(final Object that) {
-        if (this == that) {
+    public boolean equals(final Object o) {
+        if (this == o) {
             return true;
         }
-
-        if (!(that instanceof MantaMultipartUploadTuple)) {
+        if (!(o instanceof MantaMultipartUploadTuple)) {
             return false;
         }
 
-        MantaMultipartUploadTuple tuple = (MantaMultipartUploadTuple) that;
+        MantaMultipartUploadTuple tuple = (MantaMultipartUploadTuple) o;
         return partNumber == tuple.partNumber
```

Comment:
special handling for primitives where ```Objects.equals()``` isn't used. For more information, kindly visit this [page](https://stackoverflow.com/questions/5038204/apache-commons-equals-hashcode-builder)

Reply:
This handling is now consistent across the entire Java SDK.